BE/Business.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Business.Access.Layer.Helpers.GlobalExceptionHandler
+{
+    internal class AppException : Exception
```

Comment:
Class name should be more descriptive. Currently its no different from Exception class.

Reply:
I made it just for testing purposes, this is a base code for global exception handler.